specs/plugin-extensibility.md

```diff
@@ -120,7 +120,7 @@ Plugin config can be also set/overriden during signing with the `notation sign`
 
 ### Plugin contract
 
-* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`.
+* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`. The output size is limited to less than 64 MiB per channel to avoid potential plugin unexpected behavior that triggers out-of-memory issues.
```

Comment:
At line 34,
> Currently, Notation facilitates the installation of plugin executable files and archives, with a size limit of less than 256 MiB. Consequently, the plugin executable file and archive size MUST be less than 256 MiB.

Do you want to use the similar wording by adding "currently"?

```suggestion
* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`. Currently, a size limit of less than 64 MiB is applied to each output channel for preventing out-of-memory issues of potential plugin malfunctioning.
```

Reply:
Updated.